### PR TITLE
Rm atom struct

### PR DESCRIPTION
**Pull request by rmarabini**
add renumber and rename functions to atom_struct.py which allows to shift the residues  position by an offset and change the chain name

Also a have modified the format used by invert_hand protocol to print the progressbar 
since scipion uses variable fonts so the bar size changes in each print
because   the "space" width is different from the "=" width

### CODE REVIEW

File: pwem/convert/atom_struct.py [hunk 1]
```diff
@@ -689,6 +689,34 @@ def extractChain(self, chainID, start=0, end=-1,
         io.set_structure(self.structure)
         io.save(filename, sel)
 
+    def reNameChain(self, chainID, newChainName, modelID='0',
```
Comment (delarosatrevin) [inline pwem/convert/atom_struct.py:692]: Weird camelCase...what about `renameChain` ?

File: pwem/convert/atom_struct.py [hunk 2]
```diff
@@ -689,6 +689,34 @@ def extractChain(self, chainID, start=0, end=-1,
...
+                    filename="output.mmcif"):
+        self.structure[modelID][chainID].id = newChainName
+        self.write(filename)
+
+
+    def reNumberChain(self, chainID, offset=0, modelID='0',
```
Comment (delarosatrevin) [inline pwem/convert/atom_struct.py:698]: `renumberChain` ?

File: pwem/convert/atom_struct.py (hunk 1 above)
Comment (rmarabini) [reply pwem/convert/atom_struct.py]: done

File: pwem/convert/atom_struct.py (hunk 2 above)
Comment (rmarabini) [reply pwem/convert/atom_struct.py]: done

**Comment by rmarabini** [comment]
refactorization done

**Review by delarosatrevin — APPROVED** [review]

**Review by pconesa — APPROVED** [review]

**Pull request merged**